.editorconfig

```diff
@@ -9,10 +9,11 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 
-[*.{sh,xml,launch}]
```

Comment:
Why remove this?

Reply:
These use space and indent 4, so included in the general case